data/mods/PKs_Rebalance/pk_critter_misc.json

```diff
@@ -630,8 +630,7 @@
             "type": "bite",
             "cooldown": 30,
             "accuracy": 7,
-            "no_infection_chance": 3,
-            "armor_multiplier": 0.5
```

Comment:
What's wrong with this? I saw these lines in core ```monsters.json``` and ```armor_multiplier``` appears in damage-related code. Could you justify this point?